packages/cli/src/runners/task-broker.service.ts

```diff
@@ -502,4 +502,45 @@ export class TaskBroker {
 		this.pendingTaskOffers.push(offer);
 		this.settleTasks();
 	}
+
+	/**
+	 * For testing only
+	 */
```

Comment:
Not sure how I feel about adding testing specific functions

Reply:
We can add a decorator to enforce they're only called during Jest runs? :)
I find this cleaner than adding ts-ignores for private fields and methods, but open to ideas